packages/blocks/src/page-block/edgeless/edgeless-page-service.ts

```diff
@@ -54,6 +54,7 @@ export class EdgelessPageService extends PageService {
   slots = {
     edgelessToolUpdated: new Slot<EdgelessTool>(),
     pressShiftKeyUpdated: new Slot<boolean>(),
+    pressSpaceBarUpdated: new Slot<boolean>(),
```

Comment:
Thank you for your suggestion; I'll incorporate the direct assignment for the spacebar. However, I have a query regarding the shift key. Since it seems to perform a similar function to the spacebar, could we apply the same approach for the shift key? I haven't delved into the other functionalities yet, so I'm uncertain if a separate slot for the shift key is necessary.

Reply:
The shit key slot is about to be removed too. Those slots are too redundant and only increase the complexity.